packages/circuits/shasums.txt

```diff
@@ -57,8 +57,8 @@ adf1370b4a5bf3337c59883585b1a5124ac7e18518709b7c0b76293df9339a97  ./circuits/inc
 0926c062e2bcded731a8f944d0ec0cb3875868e0ab8f88371937e64dec898850  ./circuits/include/smt/smtverifierlevel.circom
 3551c671748332ba4c27c8b81fe94bad630f01a184cc9c5f6d3bc07ec6c0d1b9  ./circuits/include/smt/smtverifiersm.circom
 350c325368fa69460f2c9fc79d1928ee8f56b576c88321199e770e1a9e9d5ef1  ./circuits/include/switcher.circom
-d878fa9aee52b1137e4b639b0a840c02dd143bd6b75842d397934cc1390d7e65  ./circuits/joinsplit.circom
-641080ee9ae9739af217be47e25ff04a014f9c04c3716d6e1f1dd46303d201d1  ./circuits/lib.circom
+95723b2a56393168eb9dd8ec99b59aedea3abc18e6400b2db22b351359f3e367  ./circuits/joinsplit.circom
+faebad58ff5880abacb04174f0b1e33a982292eedee1b1f0852fd24d2294e97a  ./circuits/lib.circom
```

Comment:
This PR forgot to rerun build
https://github.com/nocturne-xyz/protocol/pull/5/files